cuda_core/tests/test_module.py

```diff
@@ -29,6 +36,14 @@
 """
 
 
+@pytest.fixture(scope="module")
+def cuda_version():
+    # binding availability depends on cuda-python version
+    _py_major_ver, _ = cuda_utils.get_binding_version()
+    _driver_ver = cuda_utils.handle_return(driver.cuDriverGetVersion())
+    return _py_major_ver, _driver_ver
```

Comment:
I think this check can be more specific
```suggestion
def is_using_old_backend():
    # binding availability depends on cuda-python version
    _py_major_ver, _ = cuda_utils.get_binding_version()
    _driver_ver = cuda_utils.handle_return(driver.cuDriverGetVersion())
    return _py_major_ver < 12 or _driver_ver < 12000
```
because both checks matters in our CI (whereas currently only the driver version is used). For example, I believe our CI  test this combo: `cuda-core` + `cuda-bindings` v11 + latest driver v12.